.gitignore

```diff
@@ -6,4 +6,7 @@ _build/
 dist/
 MANIFEST
 Axelrod.egg-info
-.hypothesis/eval_source/
+.hypothesis/
```

Comment:
To be honest, I haven't looked into hypothesis at all and simply added this to .gitignore because I kept accidentally committing test files.